sources/ProfilesColorsPreferencesViewController.m

```diff
@@ -171,6 +172,10 @@ - (void)awakeFromNib {
                     key:KEY_USE_CURSOR_GUIDE
                    type:kPreferenceInfoTypeCheckbox];
 
+    [self defineControl:_useVGuide
+                    key:KEY_USE_VERT_CURSOR_GUIDE
+                   type:kPreferenceInfoTypeCheckbox];
```

Comment:
When you rebase this will need to take an extra argument of `relatedView:nil`. That's used for searching preferences, but since this is a checkbox it won't need any more information for the search engine.

Reply:
Marking resolved for now.  I'm having trouble getting GitHub to recognize the work done in the rebase and also pull down the latest HEAD from outside my branch.